introduction.md

```diff
@@ -1 +1,5 @@
-Hi everyone, welcome to my custom dojo rules git repo ! I am a JavaScript kind of man btw
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?